### PR TITLE
Add 1.21 Docs Shadows to GitHub teams

### PR DESCRIPTION
**Pull request by reylejano**
Add 1.21 Docs Shadows to `website-milestone-maintainers` in  sig-docs/teams.yaml and `release-team` in sig-release/teams.yaml GitHub teams

/assign @palnabarun @kikisdeliveryservice @savitharaghunathan @bai

/hold for feedback if shadows should be added to the `website-milestone-maintainers` team. 
I think they should have the ability to add the 1.21 milestone on kubernetes/website PRs.

### CODE REVIEW

File: config/kubernetes/sig-release/teams.yaml
Comment (savitharaghunathan) [inline config/kubernetes/sig-release/teams.yaml:46]:
```suggestion
    + ChandaniM123 # 1.21 Docs Shadow
```

https://github.com/kubernetes/org/issues/2436 will be closed today via https://github.com/kubernetes/org/pull/2441

File: config/kubernetes/sig-release/teams.yaml
Comment (savitharaghunathan) [inline config/kubernetes/sig-release/teams.yaml:259]:
```suggestion
    + ChandaniM123 # 1.21 Docs Shadow
```

**Review by savitharaghunathan — CHANGES_REQUESTED** [review]

File: config/kubernetes/sig-release/teams.yaml
Comment (savitharaghunathan) [reply config/kubernetes/sig-release/teams.yaml:46]: Also, please swap   
   - cheftako # Cloud Provider
   - chandanim123 # 1.21 Docs Shadow

File: config/kubernetes/sig-release/teams.yaml
Comment (reylejano) [reply config/kubernetes/sig-release/teams.yaml]: Changes made and commits squashed

**Review by savitharaghunathan — COMMENTED** [review]
/lgtm

I agree with shadows having permissions to add/remove milestones. 

@reylejano please make sure that the handbook reflects this and add a cleanup step for the same. Thank you.

**Review by palnabarun — COMMENTED** [review]
> if shadows should be added to the milestone maintainers team.

@reylejano -- I think access for adding/removing milestones on `k/website` is given through the `website-milestone-maintainers` group. Ref: https://github.com/kubernetes/org/blob/master/config/kubernetes/sig-docs/teams.yaml#L342

If the shadows don't need access to globally add/remove milestones, I think `website-milestone-maintainers` is the better place.

LMK what do you think.

/hold for resolution on the above

**Comment by reylejano** [comment]
> > if shadows should be added to the milestone maintainers team.
> 
> @reylejano -- I think access for adding/removing milestones on `k/website` is given through the `website-milestone-maintainers` group. Ref: https://github.com/kubernetes/org/blob/master/config/kubernetes/sig-docs/teams.yaml#L342
> 
> If the shadows don't need access to globally add/remove milestones, I think `website-milestone-maintainers` is the better place.
> 
> LMK what do you think.
> 
> /hold for resolution on the above

@palnabarun I removed the shadows from the `milestone-maintners` and added them to `website-milestone-maintainers` in sig-docs/teams.yaml

**Comment by palnabarun** [comment]
@reylejano -- Awesome! Thank you so much :)

/lgtm
/approve

**Pull request merged**
